koku/api/report/azure/openshift/query_handler.py

```diff
@@ -53,6 +59,72 @@ def __init__(self, parameters):
         super().__init__(parameters)
         # super() needs to be called before _get_group_by is called
 
+    @property
+    def annotations(self):
+        """Create dictionary for query annotations.
+
+        Returns:
+            (Dict): query annotations dictionary
+
+        """
+        annotations = {"date": self.date_trunc("usage_start")}
+        # { query_param: database_field_name }
+        fields = self._mapper.provider_map.get("annotations")
+        for q_param, db_field in fields.items():
+            annotations[q_param] = Concat(db_field, Value(""))
+        if (
+            "project" in self.parameters.parameters.get("group_by", {})
+            or "and:project" in self.parameters.parameters.get("group_by", {})
+            or "or:project" in self.parameters.parameters.get("group_by", {})
+        ):
+            annotations["project"] = Concat("namespace", Value(""))
```

Comment:
Am I reading this right that you're concatenating an empty string to namespace?

Reply:
You know, I copied this from the ocp query handler, and didn't think twice. I will fix this to a `Coalesce` here and there.